components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -372,9 +384,134 @@ abstract class AbstractFetchDownloadService : Service() {
         context.unregisterReceiver(broadcastReceiver)
     }
 
+    @VisibleForTesting
```

Comment:
If this function is `open` should we make it `PROTECTED` instead of the default `PRIVATE`?

Reply:
Thanks, for spotting this. As I changed the structure of the code,  I do not need neither `open`, `PROTECTED` nor `PRIVATE` for testing.